doc/2/guides/main-concepts/5-authentication/index.md

```diff
@@ -2,9 +2,362 @@
+
+## `local` Strategy
+
+The `local` allows users to authenticate with a `username` and a `password`.  
+
+Thoses informations must be passed to the [auth:login](/core/2/api/controllers/auth/login) API action body:
+```bash
+kourou auth:login -a strategy=local --body '{
+  username: "mylehuong",
+  password: "password"
+}'
+```
+
+### `local` Strategy Configuration
+
+The strategy can be configured under the `plugins.kuzzle-plugin-auth-passport-local` configuration key.
+
+```js
+{
+  "plugins": {
+    // [...]
+
+    "kuzzle-plugin-auth-passport-local": {
+
+      // one of the supported encryption algorithms
+      // (run crypto.getHashes() to get the complete list).
+      "algorithm": "sha512",
+      
+      // boolean and controlling if the password is stretched or not.
+      "stretching": true,
+
+      // describes how the hashed password is stored in the database
+      // https://nodejs.org/api/buffer.html#buffer_buf_tostring_encoding_start_end
+      "digest": "hex",
+
+      // determines whether the hashing algorithm uses crypto.createHash (hash)
+      // or crypto.createHmac (hmac).
+      // https://nodejs.org/api/crypto.html
+      "encryption": "hmac",
+
+      // if true, kuzzle will refuse any credentials update or deletion,
+      // unless the currently valid password is provided
+      // or if the change is performed via the security controller
+      "requirePassword": false,
+
+      // a positive time representation of the delay after which a
+      // reset password token expires (see ms for possible formats).
+      "resetPasswordExpiresIn": -1, 
+
+      // set of additional rules to apply to users, or to groups of users
+      "passwordPolicies": []
+    }
+  }
+}
+```
+
+### Password Policies
+
+Password policies can be used to define a **set of additional rules to apply to users**, or to groups of users.
+
+Each password policy is an object with the following properties:
+
+* `appliesTo`: (mandatory). can be either set to the `*` to match all users, or an object.
+* `appliesTo.users`: an array of user `kuids` the policy applies to.
+* `appliesTo.profiles`: n array of `profile` ids the policy applies to.
+* `appliesTod.roles`: an array of `role` ids the policy applies to.
+
+::: info
+At least one of `users`, `profiles` or `roles` properties must be set if `appliesTo` is an object.
+:::
+
+### Optional properties
+
+* `expiresAfter`: a positive time representation of the delay after which a password expires (see [ms](https://www.npmjs.com/package/ms) for possible formats). Users with expired passwords are given a `resetPasswordToken` when logging in and must change their password to be allowed to log in again.
+* `forbidLoginInPassword`: if set to `true`, prevent users to use their username in part of the password. The check is case-**in**sensitive.
+* `forbidReusedPasswordCount`: the number of passwords to store in history and check against when a new password is set.
```

Comment:
```suggestion
* `forbidReusedPasswordCount`: the number of passwords to store in history and checked against when a new password is set to prevent passwords reuse.
```